<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>635</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>621</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ViewerLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QSlider" name="vertex_thicc">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>25</number>
       </property>
       <property name="sliderPosition">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="line_thicc">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="sliderPosition">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Browse">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
   </widget>
   <widget class="QLabel" name="model_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="color">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Set color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="color_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Set background color</string>
    </property>
   </widget>
   <widget class="QSlider" name="ScaleSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>621</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="line_t">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Line thickness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_line_wid">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>✖</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mUp">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>230</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mDown">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mRight">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mLeft">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rRight">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>⇢</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rDown">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>⇣</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rUp">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>⇡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rLeft">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>⇠</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gif">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Make gif</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changePerspective">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>150</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Change perspective</string>
    </property>
   </widget>
   <widget class="QPushButton" name="centerModel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>480</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Center model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showVertexes_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>210</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Change vertex display method</string>
    </property>
   </widget>
   <widget class="QPushButton" name="color_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Set vertex color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeJpeg">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>430</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>screenshot jpeg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeBmp">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>480</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>screenshot bmp</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentdir">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>540</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="translateBy">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>270</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="rotateBy">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>380</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="rzDown">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>↘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rzUp">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>↖</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mForward">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>⇧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mBackward">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>220</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>⇩</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showVertexes">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show vertexes</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="brokenLines">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>110</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Edged lines</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showLines">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show lines</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="vertex_t">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Vertex thickness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_vertex_wid">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>✖</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
